infra/base-images/base-builder/detect_repo_test.py

```diff
@@ -75,12 +100,17 @@ def check_commit_with_repo(self, repo_origin, repo_name, commit, tmp_dir):
         'python3', 'detect_repo.py', '--src_dir', tmp_dir, '--example_commit',
```

Comment:
don't pass commit if it's not specified, as you need to remove it from the new test, if I understand correctly